apps/aetx/src/aetx.erl

```diff
@@ -249,7 +249,7 @@ check_tx(#aetx{ cb = CB, tx = Tx } = AeTx, Trees, Env) ->
     end.
 
 check_minimum_fee(AeTx) ->
-    case fee(AeTx) >= aec_governance:minimum_tx_fee() of
+    case fee(AeTx) >= (min_gas(AeTx) * aec_governance:minimum_gas_price()) of
```

Comment:
I see this brings this in line with `aec_tx_pool:check_minimum_fee/2` (i.e. `aetx:fee(Tx) >= (aetx:min_gas(Tx) * aec_governance:minimum_gas_price())`). Ok.